Frontend/src/components/Connections.jsx

```
@@ -0,0 +1,87 @@
+        <input
+          type="number" // ✅ Use "number" type
+          value={friendId} // ✅ Use the correct state variable
+          onChange={(e) => setFriendId(e.target.value)} // ✅ Update state correctly
+          placeholder="Enter friend_id"
+          className="w-full p-2 border border-gray-300 rounded-lg focus:outline-none focus:ring-2 focus:ring-blue-500"
+        />
+        <button 
+          onClick={handleAddConnection} 
+          className="bg-blue-500 hover:bg-blue-700 text-white font-bold py-2 px-4 rounded-lg"
+        >
+          Add Connection
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)